lib/releases.sh

```diff
@@ -84,7 +84,7 @@ else
   # the local override created for main branch is not conflicting with existing
   # release tag. 
   export CAPM3RELEASE="v1.4.99"
```

Comment:
this has to be bumped to v1.5.99 here and in CAPM3 as well

Reply:
done